### ISSUE TITLE
adjust `aapb-collaboration-27-e` batch 

### ISSUE THREAD

**Issue by keighrim** [clamsproject/aapb-annotations#105]
As discussed in a recent meeting (and reported at https://github.com/clamsproject/aapb-collaboration/issues/27#issuecomment-2501578284), we might need to split `27-e` batch that currently contains two sets of GUIDs from "challenging images" annotation (namely `bm` set and `pbd` set) so that we can full reserve `pbd` set as the "test" set moving forward, that can be fed to scripted evaluation process without additional manual intervention. 

However, on the training side, right now I use `pbd` set as a validation set during training SWT models for 7.1 release (https://github.com/clamsproject/app-swt-detection/issues/116, no new model was trained since 7.1, current version is 7.5). And keep `27-d` batch for testing both timepoint-wise and timeframe-wise, as the `27-d` batch is annotated (unlike `pbd` set) in the "sequentially" manner that enables natural conversion from point annotations to timeframe annotations. Re-purposing `pbd` set to be the test set for the future might imply that we need[^1] either 
- a new validation set (a.k.a "dev" set that's used for evaluation-during-training for hyperparameter tuning) 
- to go back to k-fold validation 

And I want to hear any preferences, other suggestion, or any idea about this, especially @owencking and @marcverhagen. 

[^1]: https://datascience.stackexchange.com/questions/18339/why-use-both-validation-set-and-test-set

**Comment by keighrim** [comment]
In a in-person meeting today, I and @owencking decided not to split `e` batch, but instead, to develop a new "holdout" set for future model training. That said, this issue will close when a new PR that re-does #104 is merged to the main to make the sub-directory for annotations done on `e` set is fully published.
